src/index.html

```diff
@@ -1,14 +1,67 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="header">
+      <a  href="#apple.com">
+        <img src="./images/logo.png"
+          alt="images"
+          class="logo"
+        >
+      </a>
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item">
```

Comment:
you have elements of nav block, but where is nav block itself?
```suggestion
      <nav class="nav">
        <ul class="nav_list">
          <li class="nav_item">
```